targets/TARGET_NUVOTON/TARGET_NANO100/device/TOOLCHAIN_ARM_MICRO/NANO130.sct

```diff
@@ -1,27 +1,41 @@
 #! armcc -E
 
+; 512 KB APROM
+#if !defined(MBED_APP_START)
+  #define MBED_APP_START            0x00000000
+#endif
+
+#if !defined(MBED_APP_SIZE)
+  #define MBED_APP_SIZE             0x00020000
+#endif
+
+; 64 KB SRAM (internal)
+#define MBED_RAM_START              0x20000000
+#define MBED_RAM_SIZE               0x4000
+
 #if !defined(MBED_BOOT_STACK_SIZE)
   #define MBED_BOOT_STACK_SIZE      0x400
 #endif
 
-LR_IROM1 0x00000000 {
-  ER_IROM1 0x00000000 {  ; load address = execution address
+#define VECTOR_SIZE                 (4*(16 + 142))
+
+LR_IROM1 MBED_APP_START  MBED_APP_SIZE  {
+  ER_IROM1 MBED_APP_START  MBED_APP_SIZE  {  ; load address = execution address
    *(RESET, +First)
    *(InRoot$$Sections)
    .ANY (+RO)
   }
-  
-  
-  ARM_LIB_STACK 0x20000000 EMPTY MBED_BOOT_STACK_SIZE {
+
+  ARM_LIB_STACK  MBED_RAM_START  EMPTY  MBED_BOOT_STACK_SIZE  {
   }
-  
-  RW_IRAM1 AlignExpr(+0, 16) {  ; 16 byte-aligned
+
+  RW_IRAM1  AlignExpr(+0, 16)  {  ; 16 byte-aligned
    .ANY (+RW +ZI)
   }
   
-  ARM_LIB_HEAP AlignExpr(+0, 16) EMPTY (0x20000000 + 0x4000 - AlignExpr(ImageLimit(RW_IRAM1), 16)) {
+  ; Extern SRAM for HEAP
+  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_START + MBED_RAM_SIZE - AlignExpr(ImageLimit(RW_IRAM1), 16))  {
   }
 }
-ScatterAssert(LoadLimit(LR_IROM1) <= 0x00020000)    ; 128 KB APROM
-ScatterAssert(ImageLimit(ARM_LIB_HEAP) <= 0x20004000)   ; 16 KB SRAM
-
+ScatterAssert(LoadLimit(LR_IROM1) <= (MBED_APP_START + MBED_APP_SIZE))    ; 512 KB APROM
+ScatterAssert(ImageLimit(RW_IRAM1) <= (MBED_RAM_START + MBED_RAM_SIZE))   ; 64 KB SRAM (internal)
```

Comment:
Fix to 16 KB SRAM (internal)

Reply:
@ccli8 - Thanks for review. All the comments are addressed, please review once.